definitions/features/downstream.rb

```diff
@@ -16,13 +20,14 @@ def at_least_version?(version)
   end
 
   def current_version
-    @current_version ||= rpm_version('satellite') || version_from_source
+    @current_version ||= current_feature.send(:current_version)
   end
 
   def current_minor_version
     current_version.to_s[/^\d+\.\d+/]
   end
 
+  # TODO: Modify activation_key changes as per server
   def setup_repositories(version)
     activation_key = ENV['EXTERNAL_SAT_ACTIVATION_KEY']
```

Comment:
Previous [conversation](https://github.com/theforeman/foreman_maintain/commit/4af44714aae88cc5c291a5fd2d0c5d1c5d6527ae#commitcomment-33718141) erased:

> @ntkathole, @jameerpathan111,
Could confirm that does this step work same for capsule as well in case of dogfod servers?Also confirm if ENV variables are same/different for Capsule?

>@kgaikwad It will work if capsule is registered with satellite using activation key and organization.We need to consider following scenarios:
>1. Capsule registered with satellite using environment and organization
subscription-manager register --org="Default_Organization" --environment="Library"
>2. Capsule registered with satellite using activation-key and organization
subscription-manager register --org="Default_Organization" --activationkey="<Activation Key Name>"
>3. Capsule can also be registered with directly CDN.
In this case we also need to take care right red hat repos should be enabled on capsule.